api/management/commands/telegram_watcher.py

```diff
@@ -27,40 +26,47 @@ def handle(self, *args, **options):
         offset = 0
         while True:
             time.sleep(self.rest)
+            try:
+                response = self.session.get(self.updates_url, params=params).json()
+            except Exception as e:
+                with open('error.log', 'a') as f:
+                    f.write(f"Error getting updates: {e}\n{traceback.format_exc()}\n")
+                continue
 
             params = {"offset": offset + 1, "timeout": 5}
             response = self.session.get(self.updates_url, params=params).json()
-            if len(list(response["result"])) == 0:
+            if not response["result"]:
                 continue
             for result in response["result"]:
-
-                try:  # if there is no key message, skips this result.
-                    text = result["message"]["text"]
-                except Exception:
+                if not result.get("message")("text"):
                     continue
-
-                splitted_text = text.split(" ")
-                if splitted_text[0] == "/start":
-                    token = splitted_text[-1]
-                    try:
-                        profile = Profile.objects.get(telegram_token=token)
-                    except Exception:
+                if not result["message"].get("text") or not result["message"]["text"].startswith("/start"):
+                    if result["message"]["text"] == "/start":
+                        token = input("Please enter your token: ")
+                        profile = Profile.objects.filter(telegram_token=token).first()
+                        if not profile:
+                            print(f"No profile with token {token}")
+                            continue
+                if len(result["message"]["text"].split(" ")) < 2:
+                    print("Invalid format. It should be: /start <token>")
```

Comment:
These print statements do not go anywhere :D 

In order to send a message to the user, one has to send a message using something similar to what we have in the function `send_message()` in `api/messages.py`. That is, make a torified request to the telegram Endpoint with the `tg_api_key`, the `message` for the user and the user `chat_id` . 

We could also instead think how to make 'Telegram' class' `send_message()` a bit more generalist so we can simply call it right here for this special case. This case is special because we know the `chat_id`, but we do not know yet what Robot it is since the token was invalid :)